wagtail/contrib/redirects/utils.py

```diff
@@ -17,7 +17,7 @@ def write_to_file_storage(import_file, input_format):
 
 
 def get_supported_extensions():
-    return ("csv", "tsv", "xls", "xlsx")
+    return "csv", "xlsx"
```

Comment:
does this mean we will no longer support `tsv`  & `xls` exports with this change?

Reply:
imports, exports are unchanged and never supported tsv/xls in the first place.